src/index.html

```diff
@@ -6,7 +6,448 @@
+          </p>   
+          <p class="cost__amount">
+            69 999 грн
+          </p>
+        </div>
+        <div class="button">
+          <a
+            class="button__content"
+            href="index.html"
+            data-qa="card-hover"
+          >
+            КУПИТЬ
+          </a>
+        </div>
+      </div>
+      <div
+      class="card"
+      >
+        <div class="photo">
+          <img
+            class="photo__content"
+            src="images/imac.jpeg"
+            alt="photo"
+          >
+        </div>
+        <div class="text-big">
+          <h2 class="text-big__content">
+            Моноблок APPLE A1419 iMac 27&quot; Retina 5K (MNED2UA/A)
+          </h2>
+        </div>
+        <div class="text-small">
+          <p class="text-small__content">
+            Код товара: 195434
+          </p>
+        </div>
+        <div class="rate">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>   
+          <p class="feedback">
+            Отзывов: 5
+          </p>
+        </div>
+        <div class="cost">
+          <p class="cost__content">
+            Цена:
+          </p>   
+          <p class="cost__amount">
+            69 999 грн
+          </p>
+        </div>
+        <div class="button">
+          <a
+            class="button__content"
+            href="index.html"
+            data-qa="card-hover"
+          >
+            КУПИТЬ
+          </a>
+        </div>
+      </div>
+      <div
+      class="card"
+      >
+        <div class="photo">
+          <img
+            class="photo__content"
+            src="images/imac.jpeg"
+            alt="photo"
+          >
+        </div>
+        <div class="text-big">
+          <h2 class="text-big__content">
+            Моноблок APPLE A1419 iMac 27&quot; Retina 5K (MNED2UA/A)
+          </h2>
+        </div>
+        <div class="text-small">
+          <p class="text-small__content">
+            Код товара: 195434
+          </p>
+        </div>
+        <div class="rate">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>   
+          <p class="feedback">
+            Отзывов: 5
+          </p>
+        </div>
+        <div class="cost">
+          <p class="cost__content">
+            Цена:
+          </p>   
+          <p class="cost__amount">
+            69 999 грн
+          </p>
+        </div>
+        <div class="button">
+          <a
+            class="button__content"
+            href="index.html"
+            data-qa="card-hover"
+          >
+            КУПИТЬ
```

Comment:
```suggestion
            Купить
```
Don't hardcode this, use styles to uppercase